src/cli-constant.js

```diff
@@ -268,6 +285,13 @@ const minimistOptions = {
   }
 };
 
+const usageSummary = `
+Usage: prettier [optionss] [path ...]
```

Comment:
typo: optionss -> options

path -> maybe something like fileGlob?

Reply:
Good catch!

I went with `file/glob` – what do you think about that?